k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -110,6 +111,7 @@
  * it removes itself. Thus, any commands that that activity submitted are
  * removed from the queue once the activity is no longer active.
  */
+@SuppressWarnings("unchecked") // TODO change architecture to actually work with generics
```

Comment:
That is what I generally do. However, this is a warning which occurs on pretty much every call to a method from `Folder`, which is about 20 warnings in this file. I thought about this for a while, and I don't think it's possible to fix from the generic way folders work, short of getting rid of the generic return types entirely.

The point of this PR is to get to a point where warnings in MessagingController are meaningful again rather than just yellow noise on the right side, to encourage actually fixing new ones if they pop up. But for these, I don't think there is a benefit to keeping them around for this particular class.

Also, there is no way to "disable inspection locally for class" :(